dbt_platform_helper/domain/codebase.py

```diff
@@ -154,15 +154,18 @@ def build(self, app: str, codebase: str, commit: str):
 
         raise ApplicationDeploymentNotTriggered(codebase)
 
-    def deploy(self, app, env, codebase, commit):
+    def deploy(self, app, env, codebase, commit=None, ref=None):
         """Trigger a CodePipeline pipeline based deployment."""
         session = self.get_aws_session_or_abort()
 
         application = self.load_application(app, default_session=session)
         if not application.environments.get(env):
             raise ApplicationEnvironmentNotFoundException(application.name, env)
 
-        self.check_image_exists(session, application, codebase, commit)
+        image_ref = commit if commit else ref
```

Comment:
I think this changes current behaviour. Currently they specify "--commit abc123" now they would have to specify "--commit commit-abc123"
```suggestion
        image_ref = f"commit-{commit}" if commit else ref
```

Reply:
Done.